x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/pages/rule_creation/helpers.ts

```diff
@@ -612,6 +614,7 @@ export const formatAboutStepData = (
     timestamp_override: timestampOverride !== '' ? timestampOverride : undefined,
     timestamp_override_fallback_disabled: timestampOverrideFallbackDisabled,
     ...(!isEmpty(note) ? { note } : {}),
+    max_signals: maxSignals ? maxSignals : DEFAULT_MAX_SIGNALS,
```

Comment:
This default value will be different from what's used in `rule_creation_ui/pages/rule_creation/index.tsx` if the Kibana max signal setting is set. Should we align the methods used to retrieve the default max signals?

Reply:
If we end up aligning frontend/backend behavior as described in the comment above, I'll probably just end up deleting this default as we'll set it on the backend and the field is optional anyways